### PR TITLE
Width switcher v2

### PR DESCRIPTION
**Pull request by PixelJanitor**

### CODE REVIEW

File: app/templates/components/snapshot-viewer-header.hbs [hunk 1]
```diff
@@ -165,16 +165,17 @@
...
-          {{collaboration/toggle-button
-            toggleCollaborationPanel=toggleCollaborationPanel
-            isResolved=false
-            unresolvedCommentThreadCount=unresolvedCommentThreadCount
-          }}
-        {{/if}}
-
-
         <div class="btn-toolbar" data-test-snapshot-approval-button>
+          {{#if snapshot.build.isFinished}}
```
Comment (cadeParade) [inline app/templates/components/snapshot-viewer-header.hbs]: Why this line? It should have this filter much further up, in `build-container.hbs` I think.

File: app/components/snapshot-viewer-full.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ const KEYS = {
   RIGHT_ARROW: 39,
 };
 
-const GALLERY_MAP = ['base', 'diff', 'head'];
+const GALLERY_MAP = ['diff', 'base', 'head'];
```
Comment (cadeParade) [inline app/components/snapshot-viewer-full.js]: Why change the order here?

File: app/templates/components/snapshot-viewer-header.hbs (hunk 1 above)
Comment (PixelJanitor) [reply app/templates/components/snapshot-viewer-header.hbs]: Good catch. I was working from two branches and bringing these small changes in from a larger PR. Some wires must have gotten crossed. Thanks

File: app/components/snapshot-viewer-full.js (hunk 2 above)
Comment (PixelJanitor) [reply app/components/snapshot-viewer-full.js]: Mike and I have talked about this for a while now. When in full screen mode and switching from base and head you currently have to press right to go to the first item and then left to go back to the last item. This order makes more sense. You are initially presented with the dif, and then a right arrow will take you to the base, and then right again shows you the head. From there its left to go back and right to go forwards. Finally making this fix with this UI update!

**Pull request merged**